From 9840939789dec9153150e8f9ae34de4171a5c15e Mon Sep 17 00:00:00 2001
From: Wang Mingyu <wangmy@fujitsu.com>
Date: Wed, 19 Jul 2023 07:31:32 +0000
Subject: [PATCH] remove lto to fix link error of clang

error message:
| tests/tst-dlopen.p/tst-dlopen.c.o: file not recognized: file format not recognized
| clang-16: error: linker command failed with exit code 1 (use -v to see invocation)

Upstream-Status: Pending

Signed-off-by: Wang Mingyu <wangmy@fujitsu.com>
---
 meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/meson.build b/meson.build
index 6fe8045..166a15b 100644
--- a/meson.build
+++ b/meson.build
@@ -28,7 +28,6 @@ add_project_arguments(['-D_GNU_SOURCE=1',
                        '-D_TIME_BITS=64'], language : 'c')
 
 possible_cc_flags = [
-                  '-flto=auto',
                   '-ffat-lto-objects',
 		  '-fstack-protector-strong',
 		  '-funwind-tables',
-- 
2.34.1

